File: python/70 StudentToString.py
class Student:
    def __init__(self,name, clas, rollno, marks):
        self.name=name
        self.clas=clas
        self.rollno=rollno
        self.marks=marks
    def __str__(self):
        return f"Student(Name: {self.name}, Class: {self.clas}, Roll No: {self.rollno}, Marks: {self.marks})"
s1=Student("ABC",9,101,85)
print(s1)

File: java/70 StudentToString.java
package code;
class Student{
	String name;
	int age;
	Student(String name,int age){
		this.name=name;
		this.age=age;
	}
	public String toString() {
		return "Student [name=" + name + ", age=" + age + "]";
	}
}

public class Test {
	public static void main(String[] args) {
		Student s1=new Student("ABC",12);
		System.out.println(s1);
	}
}